Examples/test-suite/member_pointer.i

```diff
@@ -115,11 +115,34 @@ double (Shape::*perimetervar)(void) = &Shape::perimeter;
 PerimeterFunc_td perimetervar_td = &Shape::perimeter;
 %}
 
+/* Avoid -Werror=conversion-null error caused by passing NULL into a templated
```

Comment:
Regarding changing the testcase and having these kind of non-portable build issues, SWIG should not be causing these kind of portability issues in the generated code. This is why we choose to generate C and a vastly cutdown subset of C++. Is this problem due to templates being used in the Fortran generated code? If so, why can't the non-templated approach used in the other language modules be used?  See the "SWIG and freedom" section - http://swig.org/Doc3.0/Introduction.html#Introduction_nn13. This approach has proved to work and I would like the Fortran module to also adhere to this approach.

Reply:
I checked and changing this back to NULL from `nullptr` no longer causes an error. Looks like a moot point in this case, but I will try to make sure Fortran sticks with this level of compatibility.

Reply:
Actually it still causes an error in GCC, just not clang (my local dev complier). And it's only an error because of the -Werror flag:
```
member_pointer_wrap.cxx:1087:100: error: passing NULL to non-pointer argument 1 of "Target swig::brute_cast(const Source&) [with Target = SwigOpaqueMemFunPtr; Source = long int]" [-Werror=conversion-null]
 SWIGEXPORT SWIGEXTERN SwigOpaqueMemFunPtr const _wrap_NULLPT = brute_cast<SwigOpaqueMemFunPtr>(NULL);
```

Reply:
This ended up being an issue with the overzealous automatic "native" wrapping of variables (i.e. auto-applying `%fortranconst`). Instead of changing to null and adding flags, I'm explicitly calling`%nofortranconst NULLPT;`. After the merge we can revisit the decision to try to automatically wrap as `%fortranconst`; if we make it consistent with the other languages (which we probably should do) then this becomes a nonissue.